### PR TITLE
feat: [SRM-14862]: Fixed Infra Monitored Service creation

### PR DESCRIPTION
**Pull request by deepesh-ui**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/pages/monitored-service/components/Configurations/__tests__/Configurations.util.test.ts [hunk 1]
```diff
@@ -131,4 +131,25 @@ describe('Validate configuration tab util function', () => {
...
+      updateMonitoredService,
+      saveMonitoredService,
+      fetchMonitoredService,
+      setOverrideBlockNavigation
+    })
+    const createPayload = omit(infraMonitoredService, 'isEdit')
+    expect(updateMonitoredService).toHaveBeenCalledWith({
+      ...createPayload,
+      notificationRuleRefs: undefined,
+      environmentRef: 'EnvironmentRef101',
```
Comment (karansaraswat19) [inline src/modules/85-cv/pages/monitored-service/components/Configurations/__tests__/Configurations.util.test.ts:151]: Why are we still sending environmentRef? We can skip sending this.

Comment (deepesh-ui) [reply src/modules/85-cv/pages/monitored-service/components/Configurations/__tests__/Configurations.util.test.ts:151]: will remove as part of clean-up

**Review by karansaraswat19 — APPROVED** [review]

File: src/modules/85-cv/pages/monitored-service/components/Configurations/Configurations.tsx [hunk 2]
```diff
@@ -174,6 +174,9 @@ export default function Configurations(
         cloneTemplateValue.spec.sources['changeSources'] = defaultMonitoredService?.sources?.changeSources
         sethasTemplateChangeSourceSet(true)
       }
+      if (defaultMonitoredService?.type) {
+        cloneTemplateValue.spec['type'] = defaultMonitoredService?.type
```
Comment (pankajsingh-harness) [inline src/modules/85-cv/pages/monitored-service/components/Configurations/Configurations.tsx:178]: NIT: This conditional chaining can be removed as we are already checking for the value. 
```suggestion
        cloneTemplateValue.spec['type'] = defaultMonitoredService.type
```

**Review by pankajsingh-harness — APPROVED** [review]

**Review by harshilbits — COMMENTED** [review]
LGTM

**Pull request merged**